ref/README.md

```diff
@@ -5,7 +5,7 @@ on the machine if it is not set.  Currently the number of threads is hard-coded
 the build script when running "ctest", but users can customize it at runtime for other
 runs.
 
-## Building the kernel
+## Building the kernel (HPC systems with modules)
```

Comment:
Hi Rob,

Your presentation was amazing!   I am glad to see you are on board.  WELCOME.

The "module" utility is how we switch software and software versions.  Try:

   module avail

to see what is available on Hera or Orion.   You can query specific software to see all the versions available.  Try:

   module spider intel

to see all the versions of the Intel compilers.  This is general information for HPC machines with modules.

Machines without the utility: module, well that is a different story.  Modules sets up the paths to the utilities (for example cmake), libraries (for example netcdf), and compilers (for example gnu).  Without modules and in regard to our kernels, the user needs to setup the paths to the needed utilities/compilers/libraries with a different approach.  So without "module", to run the kernel, the user need to make sure that their desired compiler/cmake/netcdf is installed and available for use.  For example, the NOAA AMD machine uses RPM to add software.

Do you have access to Hera or Orion to try some of these commands?  The command "man module" works on Orion.

Lynd

Reply:
@LyndStringer - Perhaps rename the section "Systems that use 'modules' to manage software" to emphasize the point.  Without modules, cmake will still find the package(s) if they are installed in standard locations.  When they are not, the user will need to provide cmake hints using `CMAKE_PREFIX_PATH`.  Rob did not run into that scenario, but you did. So perhaps you could provide some verbiage to document that scenario and add it in?

Reply:
@LyndStringer  Thanks for the welcome and kudos on the presentation. I'm happy to being working with at GSL and starting to contribute! Also, thanks for the more detailed explanation on the 'module' utility.  Once I get access to Orion, I'll try running some of the 'module' commands.